<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMeasuringMethodWnd</class>
 <widget class="QDialog" name="CMeasuringMethodWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMeasuringMethodWnd</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>固定摄像头</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RBFixCamOneself">
            <property name="text">
             <string>车身自标定</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RBFixCamGroundRef">
            <property name="text">
             <string>地面参照物</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RBFixCam1D">
            <property name="text">
             <string>一维线性</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RBFixCam2D">
            <property name="text">
             <string>二维线性</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>车载摄像头</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RBMovingCamSpace">
            <property name="text">
             <string>空间标定</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RBMovingCam2DOneself">
            <property name="text">
             <string>二维本车</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RBMovingCam2DTargetVehicle">
            <property name="text">
             <string>二维目标车</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
